components/TextFields/examples/TextFieldKitchenSinkExample.swift

```diff
@@ -690,6 +690,35 @@ extension TextFieldKitchenSinkSwiftExample: UITextFieldDelegate {
     textField.resignFirstResponder()
     return false
   }
+
+  func textFieldShouldClear(_ textField: UITextField) -> Bool {
+    for controller in self.allTextFieldControllers {
+      if (textField == controller.textInput) {
+        controller.setErrorText(nil, errorAccessibilityValue: nil)
+        return true
+      }
+    }
+    return true
+  }
+
+  func textField(_ textField: UITextField,
+                 shouldChangeCharactersIn range: NSRange,
+                 replacementString string: String) -> Bool {
+    for controller in self.allTextFieldControllers {
+      if (textField == controller.textInput) {
```

Comment:
_Tip:_ Remove the braces here and throughout
```diff
- (textField == controller.textInput)
+ textfield == controller.textInput
```

Reply:
Thanks. I really dislike Swift...

Reply:
Done.